pxd.c

```diff
@@ -63,6 +63,11 @@ extern const char *gitversion;
 static dev_t pxd_major;
 static DEFINE_IDA(pxd_minor_ida);
 
+static DEFINE_MUTEX(pxd_ctl_mutex);
```

Comment:
Don't we need to destroy this at module exit?

Reply:
nope.. thats a module local... nothing to destroy